From 729368fbef095c2b74e11098ad97c71a9fff10e1 Mon Sep 17 00:00:00 2001
From: chandavath <rcrajagopal@habana.ai>
Date: Tue, 01 Oct 2024 16:55:55 +0300
Subject: [PATCH] [SW-0] gaudi3: copy the reloc bin to spsram intermediately

This patch load the DIE0 ARC1 MGMT reloc bin in DIE1 spsram.

Change-Id: I14d24bdee1ce0181e270e8546262e980bcedd85b
Signed-off-by: chandavath <rcrajagopal@habana.ai>
---

diff --git a/board/habanalabs/common/post-image-fit-arc.sh b/board/habanalabs/common/post-image-fit-arc.sh
index ded7e21..05ff8a6 100755
--- a/board/habanalabs/common/post-image-fit-arc.sh
+++ b/board/habanalabs/common/post-image-fit-arc.sh
@@ -50,8 +50,6 @@
 ITS_SED_PARAMS+="s/@ARC_MGMT_ICCM_LO@/${ARC_MGMT_ICCM_LO}/g;"
 ITS_SED_PARAMS+="s/@ARC_MGMT_OFFLOAD_HI@/${ARC_MGMT_OFFLOAD_HI}/g;"
 ITS_SED_PARAMS+="s/@ARC_MGMT_OFFLOAD_LO@/${ARC_MGMT_OFFLOAD_LO}/g;"
-ITS_SED_PARAMS+="s/@ARC_MGMT_RELOC_HI@/${ARC_MGMT_RELOC_HI}/g;"
-ITS_SED_PARAMS+="s/@ARC_MGMT_RELOC_LO@/${ARC_MGMT_RELOC_LO}/g;"
 ITS_SED_PARAMS+="s/@ARC_EVENT_DCCM_HI@/${ARC_EVENT_DCCM_HI}/g;"
 ITS_SED_PARAMS+="s/@ARC_EVENT_DCCM_LO@/${ARC_EVENT_DCCM_LO}/g;"
 ITS_SED_PARAMS+="s/@ARC_EVENT_ICCM_HI@/${ARC_EVENT_ICCM_HI}/g;"
diff --git a/board/habanalabs/gaudi3/post-image-fit.sh b/board/habanalabs/gaudi3/post-image-fit.sh
index a54c8f8..df4a976 100644
--- a/board/habanalabs/gaudi3/post-image-fit.sh
+++ b/board/habanalabs/gaudi3/post-image-fit.sh
@@ -107,3 +107,5 @@
 ITS_SED_PARAMS+="s/@MGMT_STARTADDR_LO@/${MGMT_STARTADDR_LO}/g;"
 ITS_SED_PARAMS+="s/@EVENT_STARTADDR_HI@/${ARC_EVENT_ICCM_HI}/g;"
 ITS_SED_PARAMS+="s/@EVENT_STARTADDR_LO@/${EVENT_STARTADDR_LO}/g;"
+ITS_SED_PARAMS+="s/@ARC_MGMT_RELOC_HI@/${ARC_MGMT_RELOC_TEMP_HI}/g;"
+ITS_SED_PARAMS+="s/@ARC_MGMT_RELOC_LO@/${ARC_MGMT_RELOC_TEMP_LO}/g;"
